builder/amazon/common/access_config.go

```diff
@@ -65,8 +66,10 @@ func (c *AccessConfig) Config() (*aws.Config, error) {
 // the region from the instance metadata if possible.
 func (c *AccessConfig) Region() (string, error) {
 	if c.RawRegion != "" {
-		if valid := ValidateRegion(c.RawRegion); valid == false {
-			return "", fmt.Errorf("Not a valid region: %s", c.RawRegion)
+		if c.SkipValidation == false {
```

Comment:
This should be `if !c.SkipValidation` rather than a comparison.

Also I think this can be written to be more clear; something like this:

```
valid := ValidateRegion(c.RawRegion)
if valid || c.SkipValidation {
    return c.RawRegion, nil
}
return "", fmt.Errorf(...)
```